shared/oae/api/oae.api.authentication.js

```diff
@@ -249,7 +249,7 @@ define(['exports', 'jquery', 'oae.api.config', 'oae.api.i18n', 'oae.api.user', '
         });
 
         // Submit the form
-        $($.trim(form)).appendTo('body').submit();
+        $($.trim(form)).addClass("hide").appendTo('body').submit();
```

Comment:
Can you confirm that triggering a `submit` on a hidden element isn't blocked by browsers?